SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -1830,11 +1829,12 @@ public C getConfig() {
         synchronized (campaigns) {
```

Comment:
I know this isn't your fault, but the fact we're calling this method getCampaigns, which only returns Inbox campiagns, bugs me! 

@oisin Any update on this?